de.js

```diff
@@ -177,7 +177,7 @@ settings.interface.oldBrowser Alter Server Browser
 settings.interface.rarityanim Seltenheits-Animationen ausschalten
 settings.interface.showPfp Profilbilder anzeigen
 settings.interface.showTradeMsg Trade-Nachrichten anzeigen
-settings.interface.hideNonTrade Hide Non-Tradable Skins
+settings.interface.hideNonTrade Vertecke nicht handelbare Skins
```

Comment:
line 180 should be:
settings.interface.hideNonTrade Verstecke nicht handelbare Skins